packages/react-aria-components/src/Breadcrumbs.tsx

```diff
@@ -50,7 +50,7 @@ function BreadcrumbsInner<T extends object>({props, collection, breadcrumbsRef:
     <ol
       ref={ref}
       {...filterDOMProps(props, {labelable: true})}
-      slot={props.slot}
+      slot={props.slot || undefined}
```

Comment:
what is setting props.slot to `null`? or what falsey value is coming through that isn't already `undefined`?

Reply:
This was due to the type change, where I added support for passing `null` explicitly to make local props override context completely rather than merging them. But the TS types for HTML elements only allow `string | undefined` and not `null`. 😞

Reply:
Thanks, I've run into this a few times in the TS strict upgrade, just wanted to make sure